themes/default/content/docs/concepts/inputs-outputs/_index.md

```diff
@@ -320,1662 +318,18 @@ In this case, Pulumi is taking the _output_ from one resource and using it as th
-    domainName: "example.com",
-    validationMethod: "DNS",
-});
-let certValidation = new aws.route53.Record("cert_validation", {
-    records: [
-        // Need to pass along a deep subproperty of this Output
-        certCertificate.domainValidationOptions.apply(
-            domainValidationOptions => domainValidationOptions[0].resourceRecordValue),
-    ],
-    ...
-});
-```
-
-{{% /choosable %}}
-{{% choosable language python %}}
-
-```python
-certificate = aws.acm.Certificate('cert',
-    domain_name='example.com',
-    validation_method='DNS'
-)
-
-record = aws.route53.Record('validation',
-    records=[
-        # Need to pass along a deep subproperty of this Output
-        certificate.domain_validation_options.apply(
-            lambda domain_validation_options: domain_validation_options[0]['resourceRecordValue']
-        )
-    ],
-    ...
-)
-```
-
-{{% /choosable %}}
-{{% choosable language go %}}
-
-```go
-cert, err := acm.NewCertificate(ctx, "cert", &acm.CertificateArgs{
-    DomainName:       pulumi.String("example"),
-    ValidationMethod: pulumi.String("DNS"),
-})
-if err != nil {
-    return err
-}
-
-record, err := route53.NewRecord(ctx, "validation", &route53.RecordArgs{
-    Records: pulumi.StringArray{
-        cert.DomainValidationOptions.ApplyT(func(opts []acm.CertificateDomainValidationOption) string {
-            return *opts[0].ResourceRecordValue
-        }).(pulumi.StringOutput),
-    },
-    ...
-})
-if err != nil {
-    return err
-}
-```
-
-{{% /choosable %}}
-{{% choosable language csharp %}}
-
-```csharp
-var cert = new Certificate("cert", new CertificateArgs
-{
-    DomainName = "example",
-    ValidationMethod = "DNS",
-});
-
-var record = new Record("validation", new RecordArgs
-{
-    Records = {
-        cert.DomainValidationOptions.Apply(opts => opts[0].ResourceRecordValue!)
-    },
-    ...
-});
-```
-
-{{% /choosable %}}
-{{% choosable language java %}}
-
-```java
-var cert = new Certificate("cert",
-    CertificateArgs.builder()
-        .domainName("example")
-        .validationMethod("DNS")
-        .build());
-
-var record = new Record("validation",
-    RecordArgs.builder()
-        .records(
-            cert.domainValidationOptions()
-            .applyValue(opts -> opts.get(0).resourceRecordValue().get())
-            .applyValue(String::valueOf)
-            .applyValue(List::of))
-        .build());
-```
-
-{{% /choosable %}}
-
-{{< /chooser >}}
-
-Instead, to make it easier to access simple property and array elements, an {{< pulumi-output >}} lifts the properties of the underlying value, behaving very much like an instance of it. Lift allows you to access properties and elements directly from the {{< pulumi-output >}} itself without needing {{< pulumi-apply >}}. If we return to the above example, we can now simplify it:
+All resources created by Pulumi will have properties which are returned from the cloud provider API. These values are called *outputs*.
```

Comment:
I'm not sure that I agree with this one. I almost feel like that would cause some confusion by stating that outputs are not plain values, and then right after that stating you can access their plain values.

Right after this sentence, we say:

"Outputs are values of type {{< pulumi-output >}}, which behave very much like [promises](https://en.wikipedia.org/wiki/Futures_and_promises) or [monads](https://en.wikipedia.org/wiki/Monad_(functional_programming)). Simply put, outputs are a way of representing values that are not initially known but will become available once the infrastructure resource has completed provisioning, which happens *asynchronously*.

Because outputs are asynchronous, their actual plain values are not immediately available. If you need to access an output’s plain value, you can do so using one of the following options:"